### PR TITLE
changed ui of about section

### PR DESCRIPTION
**Pull request by niladrix719**
fixes: #98 

[screen-recording (3).webm](https://user-images.githubusercontent.com/91966855/216388356-4183798c-054c-44a0-aa96-37e28c88c030.webm)

Note : i am contributing in Diversion2k23

### CODE REVIEW

**Review by arpitghura — COMMENTED** [review]
<h1>It's great having you contribute to this project</h1> Welcome to the community :nerd_face: Thanks for giving your time to contribute to <b>Typing Test</b> Project.

**Comment by arpitghura** [comment]
Hey, create an issue for this PR and also provide the details if you are contributing through any event.

File: about.html [hunk 1]
```diff
@@ -38,18 +38,18 @@
...
     </nav>
 
-        <div class="about">
-            <h1>About</h1>
+        <div class="about about-about">
+            <h1 class="about-heading">About</h1>
             <div class="subtitle">
-            <p> 
+            <p class="about-para">
             Typing Test is an open-source project created by <i>Arpit Ghura</i>.
```
Comment (arpitghura) [inline about.html:45]: Make `Typing Test` bold

**Review by arpitghura — CHANGES_REQUESTED** [review]

**Comment by niladrix719** [comment]
@arpitghura Made it bold and highlighted

**Review by arpitghura — CHANGES_REQUESTED** [review]
The page content is not fully responsive on small screen devices. 
See: 
image

make the text content to have max-width of `70-80`% according to design.

**Comment by niladrix719** [comment]
> The page content is not fully responsive on small screen devices. See: image
> 
> make the text content to have max-width of `70-80`% according to design.

made it responsive for smaller devices

**Comment by niladrix719** [comment]
@arpitghura made it responsive for smaller devices

**Review by arpitghura — APPROVED** [review]

**Pull request merged**